core/templates/dev/head/pages/exploration_editor/simple_editor_tab/SimpleEditorBodyDirective.js

```diff
@@ -32,6 +32,26 @@ oppia.directive('simpleEditorBody', [
             explorationSaveService, explorationRightsService,
             explorationWarningsService, QuestionIdService) {
           $scope.data = SimpleEditorManagerService.getData();
+          $scope.chooseQuestionType = false;
+
+          var INTERACTION_TYPES = [{
```

Comment:
Shouldn't this be part of a more global constant that other services (like StatesToQuestionsService) also use? I think we already have the concept of "interaction types currently supported by the simple editor"; we should maintain all of that stuff in one place.